server/models/User.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const UserSchema = new mongoose.Schema({
+    username: String,
+    password: String,
+    firstName: String,
+    lastName: String,
+    role: String,
+    teams: [String]
```

Comment:
@Arzulun what do you think do we approach it like this? We can also not have this here, and have a separate model for the group. Something like this seems nice:
<img width="739" alt="image" src="https://github.com/user-attachments/assets/76527d11-55f0-458c-88e4-5e74e0b18c3b">

Reply:
Yeah I agree, I think having the groups be a separate collection with the members as an array would be more efficient. We could also have each user have an array with group ids for each group they are in, that might make displaying each group a specific user is in a little bit faster instead of going through each group and looking for that specific student.